generator/src/main/java/org/corfudb/generator/distributions/Operations.java

```diff
@@ -30,12 +31,19 @@ public Operations(State state) {
                 new SnapshotTxOperation(state),
                 new SleepOperation(state),
                 new RemoveOperation(state));
+//              TODO: Fix nestedTx path to enable it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)